composer/README.md

```diff
@@ -18,4 +18,4 @@ Inside distribution/build folder will be latest composer-lib with your changes.
 Execute `npm run test` on submodule.  If you want to execute tests for every submodule execute `npm run test` in this directory.
 
 ### Generate coverage reports
-Execute `npm run test-coverage` on submodule.  If you want to one coverag report for all submodules `npm run test-coverage` in this directory.
\ No newline at end of file
+Execute `npm run test-coverage` on submodule.  If you want to one coverag report for all submodules `npm run test-coverage` in this directory.
```

Comment:
typo: coverag -> coverage

If you want *a single coverage* report for all submodules... would be better imo.

Reply:
ela will fix